### ISSUE TITLE
Update binary wheel distribution on PyPI

### ISSUE THREAD

**Issue by msanders** [autopilot-rs/autopy#18]
Note for myself to update the binary wheel distribution on PyPI / pip to include the latest changes on master. This is a bit time consuming since it currently requires me to manually test and update all my virtual machines, would like to see if it's possible to automate in the future.

**Comment by msanders** [comment]
This is still in progress, ran into another issue with the CI.

**Comment by msanders** [comment]
Blocked by https://github.com/PyO3/pyo3/issues/223
